.github/scripts/goreleaser-install.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/sh
+set -e
+
+RELEASES_URL="https://github.com/goreleaser/goreleaser/releases"
+FILE_BASENAME="goreleaser"
+
+test -z "$VERSION" && {
+    echo "Unable to get goreleaser version." >&2
+    exit 1
+}
+
+test -z "$TMPDIR" && TMPDIR="$(mktemp -d)"
+TAR_FILE="$TMPDIR/${FILE_BASENAME}_$(uname -s)_$(uname -m).tar.gz"
+export TAR_FILE
+
+(
+    echo "Downloading GoReleaser $VERSION..."
+    curl -sfLo "$TAR_FILE" \
+        "$RELEASES_URL/download/$VERSION/${FILE_BASENAME}_$(uname -s)_$(uname -m).tar.gz"
```

Comment:
If we are really doing a supply chain security, shouldn't we statically provide this, so that we are always sure it's matching what we expect? Else, someone can just hijack the $RELEASES_URL with a matching version file, and put their own malware?

Reply:
If someone is able to hijack our scripts (in git or inside the actions runner), he/she could also change a hardcoded url. When we want to be safe against an attack on goreleaser's side, we need a hermetic build or several more checks in this scripts. That's not what I want to achieve here (even when I'm not yet fine with this personally).

This PR is only meant to be a very first step. The "Supply-Chain Levels for Software Artifacts" ([SLSA](https://slsa.dev/)) describes four levels. With this PR we don't even make the first one. So there will be follow ups.